core/src/test/java/org/frankframework/stream/MessageTest.java

```diff
@@ -357,16 +358,14 @@ public void testReaderAsByteArray() throws Exception {
 	}
 
 	@Test
-	public void testStringAsStaticByteArray() throws Exception {
-		Object source = testString;
-		byte[] byteArray = Message.asByteArray(source);
+	public void testStringAsStaticByteArray() {
+		byte[] byteArray = testString.getBytes();
 		assertEquals(byteArray.length, testString.getBytes().length, "lengths differ");
 	}
 
 	@Test
-	public void testByteArrayAsStaticByteArray() throws Exception {
-		Object source = testString.getBytes();
-		byte[] byteArray = Message.asByteArray(source);
+	public void testByteArrayAsStaticByteArray() {
+		byte[] byteArray = testString.getBytes();
```

Comment:
What are you testing here in these 2 tests?

They were testing the `Message.asByteArray(source)` but that method is removed. Now there's really nothing left in this test.